config/jsdoc/api/plugins/api.js

```diff
@@ -2,19 +2,18 @@
  * Define an @api tag
  * @param {Object} dictionary The tag dictionary.
  */
-exports.defineTags = function(dictionary) {
+exports.defineTags = function (dictionary) {
```

Comment:
I think this one is going to take the most getting used to for me.  Lots of reading about it on prettier/prettier#3903 - but I still don't really get the justification.  But I'm just treating it as another example of a strong belief/convention/habit that there is really no reason to hang on to.  It is really kind of liberating to just stop caring about things like this – while still getting satisfaction knowing that the ultimate goal of improved legibility due to consistency is being achieved.

Reply:
It reminds me of this very same discussion we had years ago in OpenLayers 2. And for what it's worth, this is the setting I have definitely disabled in my project configuration, but don't really care about it any more now that I see the change here.